docs/modules/hdfs/pages/usage-guide/operations/graceful-shutdown.adoc

```diff
@@ -1,6 +1,33 @@
 = Graceful shutdown
 
-Graceful shutdown of HDFS nodes is either not supported by the product itself
-or we have not implemented it yet.
+You can configure the graceful shutdown as described in xref:concepts:operations/pod_placement.adoc[].
```

Comment:
What does graceful shutdown mean here, wait-until-all-outstanding-requests-are-serviced? Always wait out the timer before shutting down? Ensure "committed" writes are actually persisted? Ensure the standby namenode has taken over before shutting down the primary?

Why are we doing this, what would the consequences be of just SIGKILLing the daemon instead? Do the client libraries fail to retry if the node dies? Do we actually lose/corrupt data?

Those are all things I'd expect this documentation to answer.

Reply:
Sadly I don't know the answer to most of these and would need to research/read code. I'm happy to have our SME @lfrancke or @Jimvin to fill this gaps, but as discussed in Planning I wouldn't block this PR on that.

Reply:
I've added a description of graceful shutdown to the docs as well as a mention of the timeout after which the process is forcibly terminated. That should add some more context and an explanation as to the meaning.